1.4.0: Unable to use docs make target

Hi! I package this project for Arch Linux.
With 1.4.0 I am unable to use the `make -C docs` target separately:

```
make: Entering directory '/build/netavark/src/netavark/docs'
/usr/bin/mandown netavark.1.md > netavark.1
make: *** No rule to make target 'publish-crate', needed by 'docs'.  Stop.
make: *** Waiting for unfinished jobs....
make: Leaving directory '/build/netavark/src/netavark/docs'
```

--- a/Makefile
+++ b/Makefile
@@ -101,6 +101,7 @@ integration: $(CARGO_TARGET_DIR)
 validate: $(CARGO_TARGET_DIR)
 	cargo fmt --all -- --check
 	cargo clippy -p netavark -- -D warnings
+	$(MAKE) docs
 
 .PHONY: vendor-tarball
 vendor-tarball: build install.cargo-vendor-filterer

--- a/contrib/cirrus/runner.sh
+++ b/contrib/cirrus/runner.sh
@@ -39,6 +39,7 @@ _run_build_aarch64() {
 }
 
 _run_validate() {
+    make -C docs .install.mandown
     make validate
 }
 

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -3,7 +3,7 @@ DATADIR ?= ${PREFIX}/share
 MANDIR ?= $(DATADIR)/man
 MANDOWN ?= $(shell export PATH; command -v mandown)
 
-docs: $(patsubst %.md,%,$(wildcard *.md))
+docs: $(patsubst %.md,%,$(wildcard *.1.md))
 
 %.1: %.1.md
 	$(MANDOWN) $< > $@

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -1,12 +1,16 @@
 PREFIX ?= /usr/local
 DATADIR ?= ${PREFIX}/share
 MANDIR ?= $(DATADIR)/man
-MANDOWN ?= $(shell export PATH; command -v mandown)
+MANDOWN ?= mandown
 
 docs: $(patsubst %.md,%,$(wildcard *.1.md))
 
 %.1: %.1.md
 	$(MANDOWN) $< > $@
+
+.PHONY: .install.mandown
+.install.mandown:
+	cargo install mandown
 
 .PHONY: install
 install:

